tests/phpunit/api/v3/ProfileTest.php

```diff
@@ -349,7 +349,7 @@ public function testGetFieldsMembershipBatchProfile() {
       'financial_type',
     ], $result['values']['financial_type_id']['api.aliases']);
     $this->assertTrue(!array_key_exists('financial_type', $result['values']));
-    $this->assertEquals(12, $result['values']['receive_date']['type']);
+    $this->assertEquals(12, $result['values']['contribution_receive_date']['type']);
```

Comment:
@eileenmcnaughton this feels a bit odd to me maybe i haven't done something explicitly right here